src/main/java/com/checkmarx/flow/service/ThresholdValidatorImpl.java

```diff
@@ -61,16 +70,15 @@ private boolean isAllowed(ScanResults scanResults, RepoProperties repoProperties
             isAllowed = isAllowedSast(scanResults, pullRequestReport);
         }
 
-        if (isAllowed && StringUtils.containsIgnoreCase(flowProperties.getEnabledVulnerabilityScanners().toString(), ScaProperties.CONFIG_PREFIX)) {
+        if (isAllowed && scaScanner.isEnabled()) {
             isAllowed = isAllowedSca(scanResults, pullRequestReport);
```

Comment:
we can put the isenables inside the isAllowed (I think they always go together), then this function will be:
`return SastScanner.isAllowed() && scaScanner.isAllowed()`